lib/Config/Menu.php

```diff
@@ -703,6 +703,8 @@ function newsletters() {
     $data['premium_plugin_active'] = License::getLicense();
     $data['is_woocommerce_active'] = $this->woocommerce_helper->isWooCommerceActive();
     $data['is_mailpoet_update_available'] = array_key_exists(Env::$plugin_path, $this->wp->getPluginUpdates());
+    $data['subscribers_count'] = Subscriber::getTotalSubscribers();
```

Comment:
This will count subscribers on every single newsletters page request, right? I'm afraid about performance - some sites have hundreds of thousands of subscribers and that's not and easy COUNT query. At the moment we use this only for some specific pages (settings, premium). Could we count it only for the stats page here?

Reply:
I thought it can be pretty fast since that query looks rather simple (https://github.com/mailpoet/mailpoet/blob/master/lib/Models/Subscriber.php#L594). But I can change it to load only on the stats page.

Reply:
In most cases it is but COUNT in SQL databases is unfortunately not read from any "global" per-table meta value because such value would cause a bottleneck in concurrent environment. Multiple concurrent transactions can see different states of DB data and COUNT query really has to go through whole primary key index to count the rows every time (https://stackoverflow.com/questions/10976328/mysql-count-performance-on-very-big-tables). For hundreds of thousands to millions of rows this can take even a second or more, depending on the power of HW.

Reply:
Fixed by loading `subscribers_count` only for non-premium users, which should be fast, because of the 2000 limit.